MCPForUnity/UnityMcpServer~/src/port_discovery.py

```
@@ -56,22 +61,43 @@ def list_candidate_files() -> List[Path]:
+
+                    # 3. Receive framed response
+                    response_header = s.recv(8)
+                    if len(response_header) != 8:
+                        return False
+
+                    response_length = struct.unpack('>Q', response_header)[0]
+                    if response_length > 10000:  # Sanity check
+                        return False
+
+                    response = s.recv(response_length)
+                    return b'"message":"pong"' in response
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)